claat/parser/md/html.go

```diff
@@ -56,6 +65,15 @@ func isBold(hn *html.Node) bool {
 func isItalic(hn *html.Node) bool {
 	if hn.Type == html.TextNode {
 		hn = hn.Parent
+	} else if hn.DataAtom == atom.Code {
+		// Look up as many as 2 levels, to handle the case of e.g. <em><bold><code>
+		for i:= 0; i < 2; i++ {
+			hn = hn.Parent
+			if hn.DataAtom == atom.Em || hn.DataAtom == atom.I {
+				return true
+			}
+		}
+		return false
 	}
 	return hn.DataAtom == atom.Em ||
```

Comment:
Could this be moved into the `if` block on L66?

Reply:
I could replicate it there but I can't move it there - I need it here to handle non-text nodes and non-code nodes.

Reply:
Ah, I misread `else if` as `else`.